docs/cloud/recipes/multi_flow_storage.md

```diff
@@ -0,0 +1,83 @@
+
+# Multi Flow Storage
+
+This recipe is for storing multiple Flows inside a single [Docker storage](api/unreleased/environments/storage.html#docker). This is useful when users have a suite of Flows that they may deploy off of some CI/CD process or they simply want to reduce the amount of images stored in their container registry. For this recipe we are going to put two example Flows—[ETL](/core/examples/etl.html) and [Map Reduce](/core/examples/map_reduce.html)—inside of the same Docker storage.
+
+[[toc]]
+
+### Flow(s) Source
+
+```python
+from prefect import task, Flow
+
+# ETL Flow
+
+@task
+def extract():
+    return [1, 2, 3]
+
+
+@task
+def transform(data):
+    return [i * 10 for i in data]
+
+
+@task
+def load(data):
+    print("Here's your data: {}".format(data))
+
+
+with Flow("ETL") as etl_flow:
+    e = extract()
+    t = transform(e)
+    l = load(t)
+
+# Map Reduce Flow
+
+@task
+def numbers_task():
+    return [1, 2, 3]
+
+
+@task
+def map_task(x):
+    return x + 1
+
+
+@task
+def reduce_task(x):
+    return sum(x)
+
+
+with Flow("Map / Reduce 🤓") as mr_flow:
+    numbers = numbers_task()
+    first_map = map_task.map(numbers)
+    second_map = map_task.map(first_map)
+    reduction = reduce_task(second_map)
+```
+
+### Adding Flows to Storage
+
+In this code block we manually add our two Flows to the same Docker storage object. Then the storage is built a single time and that new Docker storage is assigned to both Flows. When the Flows are deployed to Prefect Cloud `build=False` so the storage is not built again.
```

Comment:
Could you call out that this pattern requires both Flows exist in the same python process?  I have another recipe for you where the flows _don't_ have to exist in the same process - do you want me to PR that separately or describe it to you offline and let you write it up?

Reply:
Let's do a separate PR